src/transform/bpcells_regress_out/config.vsh.yaml

```diff
@@ -0,0 +1,64 @@
+functionality:
+  name: bpcells_regress_out
+  namespace: "transform"
+  description: |
+    Regress out the effects of confounding variables using a linear least squares regression model with BPCells.
+  authors:
+    - __merge__: /src/authors/dorien_roosen.yaml
+      roles: [ maintainer, author ]
+    - __merge__: /src/authors/robrecht_cannoodt.yaml
+      roles: [ contributor, author ]
+    - __merge__: /src/authors/weiwei_schultz.yaml
+      roles: [ contributor ]
+  arguments:
+    - name: "--input"
+      type: file
+      description: Input h5mu file
+      direction: input
+      required: true
+      example: input.h5mu
+    - name: "--output"
+      alternatives: ["-o"]
+      type: file
+      description: Output h5mu file.
+      direction: output
+      required: true
+      default: output.h5mu
+    - name: "--output_compression"
+      type: string
+      description: The compression format to be used on the output h5mu object.
+      choices: ["gzip", "lzf"]
+      required: false
+      example: "gzip"
+    - name: "--modality"
+      type: string
+      description: "The modality to run this component on."
+      default: "rna"
```

Comment:
Should we also add an argument for:

* choosing which the input layer name (default: null → .X)
* choosing the destination layer name (default: null → .X)
* If these arguments don't exist in the scanpy regress out, maybe also add them there

Reply:
It's implemented for BPCells now. 

The scanpy regress_out method doesn't allow to specify layer, the AnnData is directly passed to sc.pp.regress_out and X is automatically chosen for regression. We could get around it by making copies of the layers and so on, should I do that?

Reply:
For scanpy functions which don't allow for this, we tend to create a new AnnData from scratch with just the information that this scanpy function needs, and then copy the output to the original AnnData to write it back to disk.

If you could, that would be great!